docs/spec/serialization.rst

```diff
@@ -52,8 +52,13 @@ You can provide a filter expression in a `DNF`_ in a format of a nested list whe
 .. code:: python
 
     prediactes = [
-        [("ColumnA", "==", 5),],
-        [("ColumnA", ">", 5), ("ColumnB", "<=", datetime.date(2021, 1, 1)),],
+        [
+            ("ColumnA", "==", 5),
+        ],
+        [
+            ("ColumnA", ">", 5),
+            ("ColumnB", "<=", datetime.date(2021, 1, 1)),
+        ],
```

Comment:
```suggestion
        [("ColumnA", "==", 5),],
        [("ColumnA", ">", 5), ("ColumnB", "<=", datetime.date(2021, 1, 1))],
```

also, `prediactes`.

Generally, can you run Ctrl-Shift-F on `,)` and `,]`, remove the comma and try again? Why are we blacking docs?

Reply:
Including the comma, excluding the indent from `.. code:: python`, this is only 74 characters wide. Why does black format this?

Reply:
This particular example is one where one should remove the trailing commas and black will do it as intended.

> Why are we blacking docs?

because we can ;)

Reply:
> Unnecessary trailing commas are removed if an expression fits in one line. This makes it 1% more likely that your line won’t exceed the allotted line length limit. Moreover, in this scenario, if you added another argument to your call, you’d probably fit it in the same line anyway. That doesn’t make diffs any larger.

https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html

Maybe `,]` don't count as trailing commas?